husky.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
   hooks: {
-    'pre-push': 'lint-staged'
+    'pre-commit': 'lint-staged'
```

Comment:
Is there a typescript-related reason that this changed? I had it set to `pre-push` so that if you need to commit some work-in-progress code, you could do so without worrying about the linter

Reply:
I didn't seem to work here previously, I can change back, no problems.